src/coreclr/inc/jiteeversionguid.h

```diff
@@ -30,12 +30,13 @@
 // NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE
 //
 //////////////////////////////////////////////////////////////////////////////////////////////////////////
+//
 
-constexpr GUID JITEEVersionIdentifier = { /* a7bb194e-4e7c-4850-af12-ea9f30ea5a13 */
-    0xa7bb194e,
-    0x4e7c,
-    0x4850,
-    {0xaf, 0x12, 0xea, 0x9f, 0x30, 0xea, 0x5a, 0x13}
+constexpr GUID JITEEVersionIdentifier = { /* {71a99045-a4fe-43b3-abd9-c1742ab020b3} */
```

Comment:
a nit: to match the output format of `uuidgen.exe -s` (and comment from https://github.com/dotnet/runtime/commit/1853e26dc75d0c6f8be6ac9b47edb31510f9178d)

```suggestion
constexpr GUID JITEEVersionIdentifier = { /* 71a99045-a4fe-43b3-abd9-c1742ab020b3 */
```

Reply:
I thought I matched the format...

I’ve got to figure out a way to make my uuid generator output in the right format.